Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml

```diff
@@ -0,0 +1,2742 @@
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "65"
+    scriptarguments:
+      interval:
+        simple: "1"
+      query_expression:
+        simple: SELECT UTF8(payload) FROM events WHERE LOGSOURCENAME(logsourceid) ILIKE '%sysmon%' AND "Image" ILIKE '%\w3wp.exe' AND "Filename" ILIKE '%.aspx%'
+      range:
+        complex:
+          root: inputs.QRadarTimeRange
+      timeout:
+        simple: "600"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": -550,
+          "y": 1730
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "34":
+    id: "34"
+    taskid: a42e7e2e-00e5-4eef-8105-bfdab412d4e2
+    type: condition
+    task:
+      id: a42e7e2e-00e5-4eef-8105-bfdab412d4e2
+      version: -1
+      name: Should continue with the investigation?
+      description: Whether to continue with the investigation or close it.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "39"
+      "Yes":
+      - "38"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -330,
+          "y": 3490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: 61a214b6-9c1b-4834-8f27-87cb8c193829
+    type: condition
+    task:
+      id: 61a214b6-9c1b-4834-8f27-87cb8c193829
+      version: -1
+      name: Should block indicators automatically?
+      description: Checks whether to block the indicators automatically.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "37"
+      "yes":
+      - "36"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.autoBlockIndicators
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -330,
+          "y": 2690
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: df110072-3316-4eb8-8e68-7325f8dc9e4a
+    type: playbook
+    task:
+      id: df110072-3316-4eb8-8e68-7325f8dc9e4a
+      version: -1
+      name: Block Indicators - Generic v3
```

Comment:
the default values for blocking is based on DBOTscore - should we change them? to the ones we extracted?

Reply:
Done